tools/labs/templates/networking/5-udp-sock/udp_sock.c

```diff
@@ -114,15 +114,13 @@ int __init my_udp_sock_init(void)
 	return 0;
 
 out_release:
-	/* TODO 1/0: cleanup socket */
```

Comment:
You should modify this so that it deletes the sock_release from the template. We should build the lab skeleton clean and runnable for the students. The students have to create the socket, so there is nothing to release.

Reply:
I don't understand.